README.md

```diff
@@ -94,41 +186,82 @@ TrajectoryConverter(input_data).write_JSON("output_file_name")
 ### Add metrics data to plot
 See the [Plots Tutorial](examples/Tutorial_plots.ipynb) for details. An overview:
 ```python
+import numpy as np
 from simulariumio import ScatterPlotData
 
+converter = TrajectoryConverter(input_data) # see above to create 
 # see Plots Tutorial for parameter details
-example_scatter_plot = ScatterPlotData(
-    title=TITLE,
-    xaxis_title=X_TITLE,
-    yaxis_title=Y_TITLE,
-    xtrace=X_VALUES,
-    ytraces=Y_VALUES,
+converter.add_plot(
+    ScatterPlotData(
+        title="Something measured",
+        xaxis_title="Time (s) or something else",
+        yaxis_title="Some measurement (units)",
+        xtrace=X_VALUES,
+        ytraces={
+            "Agent 1" : Y_VALUES,
+        },
+    )
 )
-converter = TrajectoryConverter(input_data) # see above to create converter
-converter.add_plot(example_scatter_plot, "scatter")
 converter.write_JSON("output_file_name")
 ```
-___
 
-## Installation
-**Install Requires:** 
+### Render with meshes or PDB files
+If you'd like to render agents with arbitrary meshes instead of spheres, add a `DisplayData` for each agent type you'd like to be rendered with the mesh, and specify an OBJ mesh file. Alternately, you can provide a Protein Databank (PDB) file or PDB code to render atoms for a molecule. 
 
-* Requires Python 3.7 or 3.8
+See the example notebook for the converter you are using to see how to add display data for that converter.
+```python
+display_data={
+    "A" : DisplayData(
+        name="Molecule A",
+        display_type=DISPLAY_TYPE.PDB,
+        url="https://files.rcsb.org/download/3KIN.pdb",
+        color="#0080ff",
+    ),
+    "B" : DisplayData(
+        name="Molecule B",
+        display_type=DISPLAY_TYPE.OBJ,
+        url="molecule_b.obj",
+        color="#333333",
+    ),
+}
+```
 
-* If ReaDDy trajectories will be converted, the ReaDDy python package must be installed:
-(add conda forge channel if it's not already: `conda config --add channels conda-forge`)
-`conda install -c readdy readdy`
+<br/>
 
-**Stable Release:** `pip install simulariumio`
+---
 
-**Development Head:** `pip install git+https://github.com/allen-cell-animated/simulariumio.git`
+<br/>
 
-___
+## Visualize results
+1. In a supported browser (Firefox, Chrome, or Edge), navigate to https://simularium.allencell.org/viewer.
+2. Drag the file output from SimulariumIO from your file browser onto the window or choose Load model > From your device, and select your file from the file upload dialogue.
+
```

Comment:
lines 235 - 249 seem redundant with what starts on line 25. ## Convert simulation outputs so they can be visualized in the [Simularium Viewer](https://simularium.allencell.org/).

Or am I missing something?

Reply:
That first section is meant to be a quick overview and this is more detail. But lmk if you have ideas to make that more clear

Reply:
is it possible to link down here from up there? Like have a "details" link for each of those overviews? Not a big deal either way.

Reply:
yeah I'll add some anchors https://gist.github.com/asabaylus/3071099